app/api/v2/handlers/agent_api.py

```diff
@@ -59,26 +59,32 @@ async def update_agent(self, request: web.Request):
         agent = await self.update_object(request)
         return web.json_response(agent.display)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'], summary="Update an agent, or if a existing agent match cannot be found, "
+                                                   "create one. Use the paw field in the URL to specify matching "
+                                                   "criteria and the fields from the AgentSchema in the request body"
+                                                   " to specify new field values.")
```

Comment:
For the endpoints that have url parameters, you can add something like the following to the `aiohttp_apispec.docs` decorator to describe the parameters:
```
parameters=[{
  'in': 'path',
  'name': 'paw',
  'schema': {'type': 'string'},
  'required': 'true',
  'description': 'paw of the Agent to be retrieved.'
}],
```

Reply:
I think that makes sense. I've added a version of this for every url-based parameter in these 6 endpoint entries.